### ISSUE TITLE
How to add a p2(hosted) or a p2(group) in nexus3?

### ISSUE THREAD

**Issue by Orangewithsugar** [sonatype/nexus-public#414]
Hi Nexus Team!
Have this been implemented in nexus3? I try to add it but i found it only support in nexus2 and need install plugins, but in nexus3 i didn't find any method about it.
Best regards.

**Comment by nblair** [comment]
Thanks for opening an issue @Orangewithsugar. We don't currently have support for p2 hosted or group repositories. I have tagged this as an enhancement request and we'll leave it open to track demand.

**Comment by Orangewithsugar** [comment]
OK, Thank you for taking time answer~

**Comment by Lokomofeilov** [comment]
Yes, please make it possible to host p2. This would enable many legacy projects to migrate into more modern infrastructure.

**Comment by tukatu** [comment]
Many thanks to the Sonatype team for your excellent work!
Support for p2 groups (and p2 host) would be very very appreciated!
We are using Nexus2 and miss p2 groups in Nexus3.
